kite-data/kite-data-core/src/test/java/org/kitesdk/data/spi/predicates/TestInToFromString.java

```diff
@@ -117,6 +127,16 @@ public void testMultipleIntegerValues() {
   }
 
   @Test
+  public void testMultipleIntegerValuesNormalized() {
+    // normalized predicate normalizes the string values,
+    // so they won't be sorted numerically.
+    // ensures that cases where the type may not be comparable directly
+    // that they will be consistently sorted
+    Assert.assertEquals("In#toString(INT)",
+        "123,45", Predicates.in(45,123).toNormalizedString(INT));
```

Comment:
Minor: It would be nice to have a human-friendly sort order rather than lexicographic. Not a big deal if it's a pain.